static/src/javascripts/lib/cookies.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+const getShortDomain = (): string => {
+    // Trim subdomains for prod (www.theguardian), code (m.code.dev-theguardian) and dev (dev.theguardian, m.thegulocal)
+    const domain = document.domain || '';
+    return domain.replace(/^(www|m\.code|dev|m)\./, '.');
+};
+
+const getDomainAttribute = (): string => {
+    const shortDomain = getShortDomain();
+    return shortDomain === 'localhost' ? '' : ` domain=${shortDomain};`;
+};
+
+const remove = (name: string, currentDomainOnly: boolean = false): void => {
+    // Remove cookie, implicitly using the document's domain.
+    document.cookie = `${name}=;path=/;expires=Thu, 01 Jan 1970 00:00:01 GMT;`;
+    if (!currentDomainOnly) {
+        // also remove from the short domain
+        document.cookie = `${name}=;path=/;expires=Thu, 01 Jan 1970 00:00:01 GMT; domain=${getShortDomain()};`;
+    }
+};
+
+const add = (name: string, value: string, daysToLive: ?number): void => {
+    const expires = new Date();
+
+    if (daysToLive) {
+        expires.setDate(expires.getDate() + daysToLive);
+    } else {
+        expires.setMonth(expires.getMonth() + 5);
+        expires.setDate(1);
+    }
+
+    document.cookie = `${name}=${value}; path=/; expires=${expires.toUTCString()};${getDomainAttribute()}`;
+};
+
+const cleanUp = (names: string[]): void => {
+    names.forEach(name => {
+        remove(name);
+    });
+};
+
+const addForMinutes = (
+    name: string,
+    value: string,
+    minutesToLive: number
+): void => {
+    if (minutesToLive) {
```

Comment:
According to flow this should be always defined, so maybe we could remove the check?